apps/backend/src/libs/modules/controller/base-controller.module.ts

```diff
@@ -29,6 +31,11 @@ class BaseController implements Controller {
 		this.logger.info(`${request.method.toUpperCase()} on ${request.url}`);
 
 		const handlerOptions = this.mapRequest(request);
+
+		if (request.isMultipart()) {
```

Comment:
You should create a separate plugin and inject the file to the request

Reply:
I used a plugin for injecting the file into the request, but it injects a function that returns the file object, so I need this to inject the file into the request. Or do you want me to add a plugin that will execute this function to inject the file object into the request?

Reply:
You need to create a separate plugin, that will handle file injection into the request. That plugin will register fastifyMultipart plugin, add required configuration for that and then inject the file into the request. Same way as we injected the user into the request